ogusa/parameters.py

```diff
@@ -89,17 +89,23 @@ def compute_default_params(self):
             None
 
         '''
+        # reshape lambdas
+        self.lambdas = self.lambdas.reshape(self.lambdas.shape[0], 1)
```

Comment:
@jdebacker . Why not reshape the lambdas vector to be a one-dimensional numpy array?

Reply:
This helps with broadcasting in the model.  The J = 7 was already being read in from the `default_parameters.json` as a 7x1 array.  This just ensures that any array entered is put in a Jx1 shape.